src/main/java/seedu/revision/ui/StartQuizWindow.java

```diff
@@ -255,7 +298,15 @@ public Void call() {
     private void restartQuiz() {
         answerableListPanelPlaceholder.getChildren().remove(answersGridPane.getRoot());
         fillInnerParts();
+        score1 = 0;
+        score2 = 0;
+        score3 = 0;
+        total1 = 0;
+        total2 = 0;
+        total3 = 0;
+        totalScore = 0;
         score = 0;
+        accumulatedSize = 0;
```

Comment:
If using a dictionary, can just clear the dictionary or create a new blank one. might make more sense than this.

Reply:
ok I will look into it just like the previous comment